redisgraph/query_result.py

```diff
@@ -136,10 +143,21 @@ def parse_scalar(self, cell):
             else:
                 print("Unknown boolean type\n")
 
-        elif scalar_type == ResultSetScalarTypes.PROPERTY_DOUBLE:
+        elif scalar_type == ResultSetScalarTypes.VALUE_DOUBLE:
             scalar = float(value)
 
-        elif scalar_type == ResultSetScalarTypes.PROPERTY_UNKNOWN:
+        elif scalar_type == ResultSetScalarTypes.VALUE_ARRAY:
+            scalar = value
```

Comment:
Consider renaming value to array for clarity

Reply:
done